samza-core/src/main/java/org/apache/samza/config/BlobStoreConfig.java

```diff
@@ -0,0 +1,81 @@
+package org.apache.samza.config;
+
+import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+
+/**
+ * Config related helper methods for BlobStore.
+ */
+public class BlobStoreConfig extends MapConfig {
+  private static final String STORE_PREFIX = "stores.";
+  private static final String CHANGELOG_SUFFIX = ".changelog";
+
+  public static final String DEFAULT_BLOB_STORE_STATE_BACKEND_FACTORY =
```

Comment:
1. Does this need to be here or can we remove and use class.getName() instead.
2. If it does need to be here, rename field name and remove DEFAULT_. Rename DEFAULT_STORE_STATE_BACKEND_FACTORY to KAFKA_STATE_BACKEND_FACTORY. and change DEFAULT_STORE_BACKUP_FACTORIES to refer to these two properties instead.